### ISSUE TITLE
EAM Potential

### ISSUE THREAD

**Issue by cortner** [JuliaMolSim/EmpiricalPotentials.jl#2]
The next potential to transfer from JuLIP will be EAM. Do we want this here, or should it live in a separate repo? In some ways EAM are a but more complex e.g. due to the many file formats one needs to support. At the moment JuLIP is reading in EAM files via ASE then importing the data into Julia and generating the potential.

**Comment by mfherbst** [comment]
Is it possible to do something without the python dependency (i.e. is there a simple format one could support in pure Julia). If this is not possible, then I would probably have it as a separate package to avoid the hard-coded dependency on python.

**Comment by cortner** [comment]
Yes we have a pure Julia implementation of one of the EAM formats.

**Comment by mfherbst** [comment]
Ok than my gut would be to stick with our current philosophy and keep the number of packages small (splitup can be done later). The pyhon-dependent stuff you could put into an extmodule triggered, e.g. by ASEconvert.
